src/test/java/Homework19.java

```diff
@@ -7,10 +7,10 @@
 
 import java.util.List;
 
-public class homework19 extends BaseTest{
+public class Homework19 extends BaseTest{
 
     @Test
-    public void DeletePlaylistTest() throws InterruptedException {
+    public void deletePlaylistTest() throws InterruptedException {
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.


![image](https://user-images.githubusercontent.com/104761482/216989985-a32d5711-551a-4c08-9aa2-681a9d61aa63.png)

![image](https://user-images.githubusercontent.com/104761482/216990200-a75f3afb-3cf4-47f3-8975-d230ff4fa3fb.png)